src/components/Tabs/styles.css

```diff
@@ -0,0 +1,34 @@
+:root {
+  --tab-active-underline-color: var(--color-yellow);
+  --tab-count-background: var(--color-gray-lighter);
+}
+
+.tab {
+  padding-bottom: 0.5rem;
+  display: flex;
+  font-size: var(--font-size-normal);
+  align-items: center;
+  border-bottom: 2px solid transparent;
```

Comment:
Should be `0.4rem` to match styles spec. Let's avoid using `px` in favor of `rem`.